src/CSnakes.Tests/PythonStaticGeneratorTests/FormatClassFromMethods.test_basic.approved.txt

```diff
@@ -134,7 +136,8 @@ public static class TestClassExtensions
                 using PyObject a_pyObject = PyObject.From(a)!;
                 using PyObject b_pyObject = PyObject.From(b)!;
                 using PyObject __result_pyObject = __underlyingPythonFunc.Call(a_pyObject, b_pyObject);
-                return __result_pyObject.As<double>();
+                var __return = __result_pyObject.BareImportAs<double, global::CSnakes.Runtime.Python.PyObjectImporters.Double>();
```

Comment:
Why does it need the full namespace?

Reply:
It doesn't, but good practice to avoid potential conflicts with the user project (although this is very rare).